azure_functions_worker/bindings/generic.py

```diff
@@ -52,4 +52,4 @@ def decode(cls, data: datumdef.Datum, *, trigger_metadata) -> typing.Any:
 
     @classmethod
     def has_implicit_output(cls) -> bool:
-        return False
+        return True
```

Comment:
We don't need to return False for durable?

Reply:
If we create a DurableClient converter in the library, it won't be generic anymore so it won't hit this

Reply:
We would still need to add this check for backward compatibility. If the customer has pinned the library to an older version, this would fail right?